test/sslapitest.c

```diff
@@ -13093,6 +13093,24 @@ static int test_quic_tls_early_data(void)
 }
 #endif /* !defined(OSSL_NO_USABLE_TLS1_3) */
 
+#ifdef OPENSSL_SYS_WINDOWS
+static int test_load_winstore(void)
+{
+    int ret = 0;
+    SSL_CTX *ctx = SSL_CTX_new_ex(libctx, NULL, TLS_client_method());
+
+    if (!TEST_ptr(ctx))
+        goto err;
+    if (!TEST_int_eq(SSL_CTX_load_verify_store(ctx, "org.openssl.winstore:"), 1))
```

Comment:
It seems strange to be testing this via SSL. Why not use the `OSSL_STORE_*` APIs directly? Obviously not in sslapitest.c though - perhaps ossl_store_test.c might be a more obvious place.